### DISCUSSION TITLE
Docker version update

### DISCUSSION THREAD

**Discussion by P1c1s** [FreshRSS/FreshRSS#6007] [Q&A]
Hi, I discovered recently this awesome project, it's really cool. I installed FreshRss with docker  but I don't how update to correct version, at moment I have the 1.22 version. Thanks in advanced

**Comment by Alkarex** [comment]
Hello,
See our Docker documentation: https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md

In particular https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#docker-compose

If you do not do so already, I recommend you to use Docker Compose, tracking our image `freshrss/freshss` or one of its variants.

Then the key steps for updating to a new versions are:

```sh
docker-compose pull
docker-compose up -d
```
